filament/backend/CMakeLists.txt

```diff
@@ -450,3 +450,19 @@ if (APPLE AND NOT IOS)
     set_target_properties(backend_test_mac PROPERTIES FOLDER Tests)
 endif()
 
+# ==================================================================================================
+# Compute tests
+
+add_executable(compute_test
```

Comment:
Could we put these tests under the same `backend_test` target?

Reply:
they're more simple and have to run on Android -- backend_test seemed to do too much.